SECURITY.md

```diff
@@ -6,6 +6,8 @@ Patches for **Events Manager** in this repository will only be applied to the fo
 
 | Version | Supported          |
 | ------- | ------------------ |
+| 3.6.0   | :white_check_mark: |
```

Comment:
Need version update in events.yaml file

Reply:
The events manager has no events.yaml file.

Reply:
oops my bad..